README.md

```diff
@@ -12,17 +12,18 @@ Installation
 ======
 1. Install npm: `curl http://npmjs.org/install.sh | sh`
 2. Grab this module from npm: `npm install goodreads`
-3. Include it in your program:
+3. Compile Coffeescript into JavaScript files using `npm run build`
```

Comment:
If you don't want to include in this PR. just open an issue and link to it in the thread.